src/main/scala/cromwell/binding/types/WdlStringType.scala

```diff
@@ -21,6 +21,7 @@ case object WdlStringType extends WdlPrimitiveType {
 
   override def add(rhs: WdlType): Try[WdlType] = rhs match {
     case WdlStringType | WdlIntegerType | WdlFloatType => Success(WdlStringType)
+    case WdlFileType => Success(WdlFileType)
```

Comment:
This generates a WdlString presumably?

"--file=/bin/bash" is not a valid path?